app/views/tablescapes/search_results.html.erb

```diff
@@ -7,7 +7,7 @@
           <% @tablescapes.each do |t| %>
             <div class="searchcard">
               <% if t.photos.attached? %>
-                <%= cl_image_tag photo.key, class: "card-img-top" %>
+                <%= cl_image_tag t.photos[0].key, class: "card-img-top" %>
```

Comment:
this is going to clash with my latest pull request, that has added an image-carousel inside the if statement